docs/akkudoktoreos/introduction.md

```diff
@@ -0,0 +1,165 @@
+Alternatively your own collected historical data could be used to reflect your personal
+consumption behaviour.
+
+## Simulation Results
+
+Based on the input data, the EOS uses a genetic algorithm to create a cost-optimized
+schedule for the coming hours from numerous simulations of the overall system.
+
+The plan created contains for each of the coming hours
+
+- Control information
+  - whether and with what power the battery storage should be charged from the grid
+  - when the battery storage should be charged via the PV system
+  - whether discharging the battery storage is permitted or not
+  - when and with what power the electric car should be charged
+  - when a household appliance should be activated
+- Energy history information
+  - Total load of the house
+  - Grid consumption
+  - Feed-in
+  - Load of the planned household appliances
+  - Charge level of the battery storage
+  - Charge level of the electric car
+  - Active losses
+- Cost information
+  - Revenue per hour (when fed into the grid)
+  - Total costs per hour (when drawn from the grid)
+  - Overall balance (revenue-costs)
+  - Cost development
+
+If required, the simulation result can also be created and downloaded in graphical
+form as a PDF from EOS.
+
+## Integration
+
+The Akkudoktor EOS can be integrated into a wide variety of systems with a variety
+of components.
+
+![Integration](../_static/introduction/integration.png)
+
+However, the components are not integrated by the EOS itself, but must be intergrated by
+the owner using an integration component and currently requires some effort and technical
+know-how.
+
+Any [integration](#integration-page) solution that can act as an intermediary between the
+components and the REST API of EOS can be used. One possible solution that enables the
+integration of components and EOS is Node-RED. Another solution could be Home Assistant
+itself.
+
+Access to the data and functions of the components can be done in a variety of ways.
+Node-RED offers a large number of types of nodes that allow access via the protocols
+commonly used in this area, such as Modbus or MQTT. Access to any existing databases,
+such as InfluxDB or PostgreSQL, is also possible via nodes provided by Node-RED.
+
+It becomes easier if a smart home solution like Homa Assistant, openHAB or ioBroker or
+solutions such as evcc or openWB are already in use. In this case, these solutions
+already take over the technical integration and communication with the components at a
+technical level and Node-RED offers nodes for accessing these solutions, so that the
+corresponding sources can be easily integrated into a flow.
+
+In Home Assistant you could use an automation to prepare the input payload for EOS and
+then use the RESTful integration to call EOS. Based on this concept there is already a
+solutuion created by [Duetting](#duetting-solution).
+
+The plan created by EOS must also be executed via this integration component, with the
```

Comment:
New term "integration component" - what is the difference to "integration solution" and "integration"?